testimony/__init__.py

```diff
@@ -444,6 +448,90 @@ def validate_docstring_report(testcases):
         return -1
 
 
+def validate_values_report(testcases):
+    """Check that given token contains one of allowed values."""
+    result = {}
+    invalid_docstring_count = 0
+    invalid_token_value_count = 0
+    missing_docstring_count = 0
+    missing_token_count = 0
+    testcase_count = 0
+    for path, tests in testcases.items():
+        testcase_count += len(tests)
+        for testcase in tests:
+            issues = []
+            for token in SETTINGS['tokens']:
+                token.lower()
```

Comment:
`''.lower()` returns transformed string, but doesn't transform it in-place. You meant `token = token.lower()`?

Reply:
Fixed, thank you